pkgs/development/python-modules/funsor/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, makefun
+, multipledispatch
+, numpy
+, opt-einsum
+, typing-extensions
+, pyro-ppl
+, torch
+, pandas
+, pillow
+, pyro-api
+, pytest-xdist
+, pytestCheckHook
+, requests
+, scipy
+, torchvision
+}:
+
+let
+  self = buildPythonPackage rec {
+    pname = "funsor";
+    version = "0.4.5";
+
+    disabled = pythonOlder "3.7";
+
+    src = fetchFromGitHub {
+      owner = "pyro-ppl";
+      repo = "funsor";
+      rev = "refs/tags/${version}";
+      hash = "sha256-3JxnRlJN/kYstcIKt8cvDmqnp8oyoA4jkMBDVuAMa5Q=";
+    };
+
+    propagatedBuildInputs = [
+      makefun
+      multipledispatch
+      numpy
+      opt-einsum
+      typing-extensions
+    ];
+
+    nativeCheckInputs = [
+      # Backend
+      pyro-ppl
+      torch
+
+      pandas
+      pillow
+      pyro-api
+      pytest-xdist
+      pytestCheckHook
+      requests
+      scipy
+      torchvision
+    ];
+
+    doCheck = true;
```

Comment:
why is this necessary? isn't this the default?

Reply:
You are right ! Removed it :)